File: src/targets/clojure/clj_http/fixtures/headers.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/get "https://httpbin.org/headers" {:headers {:x-foo "Bar"
                                                     :x-bar "Foo"}
                                           :accept :json})
=======
(client/get "http://mockbin.com/har" {:headers {:x-foo "Bar"
                                                :quoted-value "\"quoted\" 'string'"}
                                      :accept :json})
>>>>>>> upstream/master

File: src/targets/php/guzzle/fixtures/headers.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'https://httpbin.org/headers', [
  'headers' => [
    'accept' => 'application/json',
    'x-bar' => 'Foo',
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'http://mockbin.com/har', [
  'headers' => [
    'accept' => 'application/json',
    'quoted-value' => '"quoted" \'string\'',
>>>>>>> upstream/master
    'x-foo' => 'Bar',
  ],
]);

echo $response->getBody();